Makefile.am

```diff
@@ -40,6 +40,10 @@ linkback = $(top_builddir)/src/libfabric.la
 bin_PROGRAMS = \
 	util/fi_info
 
+bin_SCRIPTS =
+
+nodist_DIR =
```

Comment:
I'm not sure that `nodist_DIR` makes sense (i.e., I don't think that automake supports a `DIR` variable to be `nodist`ed).  Do you have a companion commit over in the Cray/GNI development repository that I can look at to see how you're using this?  I have different concerns about dropping `dist_` from `dist_bin_SCRIPTS` that will probably also be cleared up by seeing the companion commit.